dash/development/component_generator.py

```diff
@@ -22,7 +26,28 @@ class _CombinedFormatter(argparse.ArgumentDefaultsHelpFormatter,
 
 # pylint: disable=too-many-locals
 def generate_components(components_source, project_shortname,
-                        package_info_filename='package.json'):
+                        package_info_filename='package.json',
+                        generate_r_components=False,
+                        rprefix=None):
+    project_shortname = project_shortname.replace('-', '_').rstrip('/\\')
```

Comment:
how was this handled before?

Reply:
The `generate_r_components` parameter no longer exists, now using `--r-prefix` as flag to set R component/package generation.